### DISCUSSION TITLE
Paperback read progress tracking integration

### DISCUSSION THREAD

**Discussion by mgrimace** [gotson/komga#1205] [Q&A]
Hello,

The documentation reads that to enable read progress tracking to install the Paperback tracker from `https://paperback-ios.github.io/trackers-main/main/` however, the Paperback tracker option there only allows 'update' that over-writes the default Paperback extension (1.2.10) with a lower number version (1.0.3, built for 0.7). Is this still necessary to do for tracking?

**Comment by gotson** [comment]
You need to ask this on the Paperback community.

**Reply by mgrimace** [reply]
Will do, for reference I’m referring to the komga documentation: https://komga.org/docs/guides/paperback
